config/model_configs/box_density_current_test.yml

```diff
@@ -11,8 +11,8 @@ z_stretch: false
 x_elem: 45
 ode_algo: "SSP33ShuOsher"
 config: "box"
-kappa_4_vorticity: 1.0e7
-kappa_4_tracer: 1.0e7
+kappa_4_vorticity: 0.2
```

Comment:
Is it clear why this is so much smaller than the other coefficients?

Reply:
@akshaysridhar The paper you posted gives ~ 3e7 m4/s for this case I think, instead of the previous 1e7. I'm going to use the default coefficient here.